.gitmodules

```diff
@@ -42,3 +42,6 @@
 	path = third-party/wlr-protocols
 	url = https://gitlab.freedesktop.org/wlroots/wlr-protocols
 	branch = master
+[submodule "third-party/jwt-cpp"]
```

Comment:
Is this packaged for the distros we support? I think we'll want the option for downstream packagers to supply their own package instead of using our submodules (like we do with other unmodified dependencies).

Reply:
I'm not sure this one is popular enough yet. Looks like they're in vcpkg though.

https://github.com/Thalhammer/jwt-cpp/blob/master/docs/install.md

@TheElixZammuto Could you make make this alphabetical in the `.gitmodules` file?